client/test/filters/FilterComponent.spec.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Checkbox } from 'react-bootstrap';
+import { mount, shallow } from 'enzyme';
+import Filter from '../../app/components/calendar/filters/Filter';
+import chai from 'chai';
+import sinon from 'sinon';
+import ConferenceRoom from '../factories/ConferenceRoom';
+
+const expect = chai.expect;
+
+describe('<Filter />', () => {
+  const color = '#000000';
+  const title = 'sample_title';
+  const id = 1;
+  const capacity = 1;
+  const conferenceRoom = ConferenceRoom.build({ id, title, color, capacity });
```

Comment:
You can use the default values of the factory, no need to override attributes :)